### ISSUE TITLE
Making code citable

### ISSUE THREAD

**Issue by JacquesCarette** [JacquesCarette/Drasil#1153]
Here are some detailed instructions on how to get a DOI for code!
https://guides.github.com/activities/citable-code/

**Comment by smiths** [comment]
I've gone through the instructions for Zenodo.  They are fairly straightforward.  Since I do not own the Drasil repo, I couldn't do it for Drasil; I followed the instructions for the Case Studies repo.

@JacquesCarette will need to follow the steps for Drasil.  (Be careful to do them in order and not accidentally skip a step like I did.)  :smile:  

Zenodo can create a DOI for a specific version of the software.  When we follow the process with make an archive of the code a specific point in time.  We'll want to give it a version number.  (Do we already have that?)  We could wait for the next significant milestone, but I think we'll always be waiting for the next milestone.  Maybe we should just archive what we have today.  If we don't have a better release number, we could use v0.1.  (We just discussed this over Zoom.)

For the Case studies repo the zenodo record is at:

https://zenodo.org/record/4521371#.YCG9u49KhQ8

The BibTeX entry is:

```
@software{spencer_smith_2021_4521371,
  author       = {Spencer Smith},
  title        = {{smiths/caseStudies: Case study examples used to 
                   gain insight into scientific software, basis for
                   Drasil case studies}},
  month        = feb,
  year         = 2021,
  publisher    = {Zenodo},
  version      = {v1.0.1},
  doi          = {10.5281/zenodo.4521371},
  url          = {https://doi.org/10.5281/zenodo.4521371}
}
```

My version 1 is cited as DOI: 10.5281/zenodo.4521371

To cite all versions you use DOI: 10.5281/zenodo.4521370
